lib/launcher.js

```diff
@@ -43,6 +43,8 @@ class Launcher {
         const packageJSON = JSON.parse(JSON.stringify(packageJSONTemplate))
         packageJSON.dependencies = JSON.parse(JSON.stringify(this.snapshot.modules))
         await fs.writeFile(this.files.packageJSON, JSON.stringify(packageJSON, ' ', 2))
+        await fs.unlink(path.join(this.projectDir, '.config.nodes.json'))
+        await fs.unlink(path.join(this.projectDir, '.config.nodes.json.backup'))
```

Comment:
May be better to use `fs.rm( path, { recursive: true } )` so it copes with them files not existing.

Reply:
Update pushed, I think it should be `{ force: true }` not `{ recursive: true }`